dts/arm/nxp/nxp_rt1060.dtsi

```diff
@@ -50,7 +50,7 @@
 		/* i.MX rt1060 has a second Ethernet controller. */
 		enet2: ethernet@402d4000 {
 			compatible = "nxp,enet";
-			reg = <0x402D8000 0x628>;
+			reg = <0x402D4000 0x628>;
```

Comment:
```suggestion
			reg = <0x402d4000 0x628>;
```

This should be lowercase to keep with [devicetree styling guidelines](https://docs.kernel.org/devicetree/bindings/dts-coding-style.html):
> Hex values in properties, e.g. “reg”, shall use lowercase hex. The address part can be padded with leading zeros.